### ISSUE TITLE
Stop printing to STDERR in unit tests

### ISSUE THREAD

**Issue by jamesaoverton** [ontodev/robot#407]
When piping `mvn` STDOUT to a file, I'm seeing unexpected messages in my terminal. I think one of the query unit tests is printing to STDERR.

```
$ mvn package > package.txt
                                                                                                                                                                  
s,p,o                                                                                                                                                               
https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl,http://www.w3.org/1999/02/22-rdf-syntax-ns#type,http://www.w3.org/2002/07/owl#Ontology    
https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test2,http://www.w3.org/2000/01/rdf-schema#subClassOf,https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test1                                                                                                                           
https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test2,http://www.w3.org/1999/02/22-rdf-syntax-ns#type,http://www.w3.org/2002/07/owl#Class
https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test1,http://www.w3.org/1999/02/22-rdf-syntax-ns#type,http://www.w3.org/2002/07/owl#Class
https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test1,http://www.w3.org/2000/01/rdf-schema#label,test one
https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test1,http://www.w3.org/2000/01/rdf-schema#label,Test 1
```

**Comment by rctauber** [comment]
This is coming from the `execVerify(Map<File, Tuple>)` method. It automatically writes to STDERR. I could overload this method, or just change to not write to STDERR it as it looks like it's only used for testing...

**Comment by jamesaoverton** [comment]
Closed by #415
